src/code/reducers/index.js

```diff
@@ -31,6 +31,45 @@ function initialState() {
           });
 }
 
+function fertilize(state) {
+  let chromosomes0 = new BioLogica.Organism(BioLogica.Species.Drake, state.drakes[0].alleleString, state.drakes[0].sex).getGenotype().chromosomes,
+      chromosomes1 = new BioLogica.Organism(BioLogica.Species.Drake, state.drakes[1].alleleString, state.drakes[1].sex).getGenotype().chromosomes,
+      alleleString = "",
+      sex = 1,
+      side, chromosome, name;
+  for (name in chromosomes0) {
+    side = motherCurrentGamete(state.gametes)[name];
+    chromosome = chromosomes0[name][side];
+    if (chromosome && chromosome.alleles) alleleString += "a:" + chromosome.alleles.join(",a:") + ",";
+  }
+  for (name in chromosomes1) {
+    side = fatherCurrentGamete(state.gametes)[name];
+    if (side === "y") sex = 0;
+    chromosome = chromosomes1[name][side];
+    if (chromosome && chromosome.alleles && chromosome.alleles.length) alleleString += "b:" + chromosome.alleles.join(",b:") + ",";
+  }
+
+  return state.setIn(["drakes", 2], {
+    alleleString,
+    sex
+  });
+}
+
+function breedClutch(state, clutchSize) {
```

Comment:
@sfentress clutch breeding is much simpler now, and uses crossover! This should simplify correct move calculations